component-server-parent/component-server-model/src/main/java/org/talend/sdk/component/server/front/model/ComponentId.java

```diff
@@ -35,4 +35,6 @@ public class ComponentId {
     private String family;
 
     private String name;
+
+    private String databaseMapping;
```

Comment:
why do we call it so specific: "databaseMapping"?
Could it be typeMapping for more general use? (I'm searching for use-cases to understand what's behind it)

Reply:
No, you should consider the studio's db mapping file like this one :
https://github.com/Talend/tcommon-studio-se/blob/9c89f86a1b094d789ebe068f7aa63f534[…]n/plugins/org.talend.core.runtime/mappings/mapping_Postgres.xml

This feature is in two parts, framework that allows to specify eventual db mapping according the studio mappings and the tup integration.
Giving the mapping allow to acccess the schema editors' db column type column .
spreading it via the SchemaInfo, the guess schema or directly in the editor.
Thus, this is mainly about schema editor access.